api/user.py

```diff
@@ -1,7 +1,14 @@
 from flask_login import UserMixin
 
+from enum import IntEnum
+
 import faf.db as db
 
+class UserGroup(IntEnum):
+    NONE = 0,
+    MODERATOR = 1,
+    ADMIN = 2
```

Comment:
Well, the group column name was there already, I'm not going to change the mysql schema. The rest I'll do.